recipes/dnp-binstrings/meta.yaml

```diff
@@ -0,0 +1,33 @@
+package:
+  name: binstrings
+  version: "1.0"
+
+source:
+  url: https://github.com/erinijapranckeviciene/dnpatterntools/files/2431226/v1.0.tar.gz
+  sha256: bb38cf23132445d9c2ffec04229a578207eb932c38a5cfc8168d5c9a79f94a39
+
+build:
+  number: 1
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+
+  host:
+    - svn
```

Comment:
if you depend on seqan you can hopefully get rid of the svn dependency

Reply:
Yes, I agree the dependences would be better. Tidier. Thank you.